<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HighScore</class>
 <widget class="QMainWindow" name="HighScore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HighScore</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>218</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>1</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>351</width>
      <height>0</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>1</width>
      <height>1</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>355</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionPong2"/>
    <addaction name="actionBibleQuiz"/>
    <addaction name="actionRedSquare"/>
    <addaction name="actionSpaceBubble"/>
    <addaction name="actionTowerDefense"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuActions"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPong2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pong2</string>
   </property>
  </action>
  <action name="actionBibleQuiz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BibleQuiz</string>
   </property>
  </action>
  <action name="actionRedSquare">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RedSquare</string>
   </property>
  </action>
  <action name="actionSpaceBubble">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SpaceBubble</string>
   </property>
  </action>
  <action name="actionTowerDefense">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>TowerDefense</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
